config-linux.md

```diff
@@ -189,6 +189,96 @@ In addition to any devices configured with this setting, the runtime MUST also s
 * [`/dev/ptmx`][pts.4].
   A [bind-mount or symlink of the container's `/dev/pts/ptmx`][devpts].
 
+## <a name="configLinuxNetworkDevices" />Network Devices
+
+Linux network devices are entities that send and receive data packets.
+They are not represented as files in the /dev directory, unlike block devices, network devices are represented with the [`net_device`][net_device] data structure in the Linux kernel.
+Network devices have their own network namespace and a set of operations distinct from regular file operations. Examples of network devices include Ethernet cards, loopback devices, and virtual devices like bridges, VLANs, and MACVLANs.
+
+This schema focuses solely on moving existing network devices identified by name into the container namespace. It does not cover the complexities of network device creation or network configuration, such as IP address assignment, routing, and DNS setup.
+
+**`netDevices`** (object, OPTIONAL) set of network devices that MUST be available in the container. The runtime is responsible for providing these devices; the underlying mechanism is implementation-defined.
+
+The runtime MUST check that is possible to move the network interface to the container namespace and MUST [generate an error](runtime.md#errors) if the check fails.
+
+The runtime MUST set the network device state to "up" after moving it to the network namespace to allow the container to send and receive network traffic through that device.
+
+Notice that after deleting a network namespace, all its migratable network devices are moved to the default network namespace, virtual devices (veth, macvlan, ...) are destroyed.
+The runtime MAY decide to move back or destroy the network device before the network namespace is deleted. If the network device is moved back, the runtime MUST set its state to "down" before moving it back to ensure that the interface is no longer active and won't interfere with other network operations or cause IP address conflicts.
```

Comment:
You are the experts in the runtime worlds so I will follow your advice, if you prefer and agree it should be MUST I don't have more argument than I'm suggesting MAY because I always prefer to be conservative when defining APIs and spec,  I'm worried if some runtimes may not be able to participate in deletion or it can be very complex for them to do it, like I imaging can be the case for kata and gvisor. Being flexible allows them to implement the spec and be fully complaint instead of having to document it as limitation.

Reply:
I think describing the current flow for intended use cases will help inform what we should do here. Are the interfaces that are moved into the containers reused or destroyed or both are valid cases? 
We could have a cleanup policy field for each device if we need to support both.

Reply:
put it that way, I think that should be a MUST, since a virtual interface has to be created out of this flow, does not make sense to add the responsability to destroy it to the runtime ... it also makes more clear the spec, with @mrunalp questions I recognize there is a lot of ambiguity, and despite it can be more amenable for runtime implementers it sounds like it be more confusing for users.

I will push a change to make this a MUST

Reply:
@mrunalp @akerouanton added a last commit with MUST return back the interface in down state https://github.com/opencontainers/runtime-spec/pull/1271/commits/47c4d3c2a4d97b7f24b62a804fe0ec6e09f30b50 , so it removes ambiguity and makes simpler to build automation over it, 

1. node has interface A or user creates interface A
2. user tells runtime "run with --net-device A"
3. user tells runtime "stop container"
4. node has interface A in down state

then is up to the user to do what it wants with interface A, if already exist it can not do much, if the user created the interface then is on him to delete or reuse it